packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -832,6 +842,7 @@ export class SnapController extends BaseController<
     encryptor,
     getMnemonic,
     getFeatureFlags = () => ({}),
+    clientCryptography = {},
```

Comment:
```suggestion
    clientCryptography,
```
Maybe just leave this as undefined?